proto/panacea/datapool/v2/pool.proto

```diff
@@ -60,9 +61,11 @@ message PoolParams {
   cosmos.base.v1beta1.Coin nft_price = 4;               // A price of a NFT
   repeated string          trusted_data_validators = 5; // Trusted data validators' addresses
   repeated string          trusted_data_issuers = 6;    // Optional. Data issuers that is trusted by the curator
-  google.protobuf.Duration download_period = 7 [
-    (gogoproto.stdduration) = true
-  ];                                                    // A download period after redeeming NFT
+  string                   curator_commission_rate = 7 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
+    (gogoproto.moretags) = "yaml:\"curator_commission_rate\"",
+    (gogoproto.nullable) = false
+  ]; // curator commission rate
```

Comment:
The unused `download_period` field has been removed, and a fixed value, `curator_commission_rate`, has been added to this Params.

Reply:
Good.
The `PoolParams` was intended to be a set of params given by user. So, would you store this `curator_commission_rate` under `Pool`, not `PoolParams`?

Reply:
OK, That looks good.
I will change this code soon.

Reply:
FIxed it. https://github.com/medibloc/panacea-core/pull/320/commits/9a32cf6431630405f85e9ceaf84c83c72b990862